Specs/Scene/Model/InstancingPipelineStageSpec.js

```diff
@@ -425,6 +427,58 @@ describe(
       });
     });
 
+    it("dequantizes normalized rotations", function () {
```

Comment:
Looks like this test is failing in Travis:

`DeveloperError: IndexDatatype.UNSIGNED_INT requires OES_element_index_uint, which is not supported on this system.  Check context.elementIndexUint.`

As with other instancing-related specs, I probably just need to skip this test in Travis